.github/workflows/release.yml

```diff
@@ -19,13 +19,6 @@ jobs:
           go-version: 1.18
       - name: Install Carvel Tools
         uses: vmware-tanzu/carvel-setup-action@v1
-        with:
```

Comment:
I think we have so far been very explicit about what versions we are releasing with. 

Our testing workflow does not float [see here](https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/.github/workflows/test-gh.yml#L28-L35). Keeping these in sync very explicitly means we know that the tested dependencies matches the released dependencies.

Reply:
Got it. Using specific versions only now.